README.md

```diff
@@ -371,7 +371,7 @@ redis = Redis.new(
 [ghostunnel]: https://github.com/square/ghostunnel
 [OpenSSL::SSL::SSLContext documentation]: http://ruby-doc.org/stdlib-2.3.0/libdoc/openssl/rdoc/OpenSSL/SSL/SSLContext.html
 
-*NOTE:* SSL is only supported by the default "Ruby" driver
+*NOTE:* SSL is only supported by the default "Ruby" driver and [hiredis](https://github.com/redis/hiredis)
```

Comment:
It seems that it is not enough to use the feature yet. Should we wait for to be able to use the feature of hiredis-rb?

https://github.com/redis/redis-rb/blob/610c783e48b12a795e0fb6c1c5bfe0adce572dc7/lib/redis/connection/hiredis.rb#L12-L27

* https://github.com/redis/hiredis-rb/issues/58
* https://github.com/redis/hiredis-rb/pull/72